documentation/asciidoc/topics/proc_exposing_loadbalancer.adoc

```diff
@@ -14,7 +14,10 @@ a load balancer service.
 
 . Include `spec.expose` in your `Infinispan` CR.
 . Specify `LoadBalancer` as the service type with the `spec.expose.type` field.
-. Optionally specify a node port to which the load balancer forwards traffic with the `spec.expose.nodePort` field.
+. Configure ports with the following fields if appropriate:
++
+* `spec.expose.port` specifies the network port where the service is exposed. The default port is `7900`.
+* `spec.expose.nodePort` optionally specifies a node port to which the load balancer forwards traffic.
```

Comment:
`nodePort` is not used by `LoadBalancer` service type. 
Source Code: https://github.com/infinispan/infinispan-operator/blob/main/pkg/controller/infinispan/resources/service/service_controller.go#L382

Reply:
great. that answers it. thanks @pruivo ! I know it seems weird and was looking back through...